src/operations/write.ts

```diff
@@ -65,7 +65,8 @@ export const write = (
   if (operation.operation === 'query') {
     writeSelection(ctx, 'Query', select, data);
   } else {
-    writeRoot(ctx, select, data);
+    const isMutation = operation.operation === 'mutation';
```

Comment:
I know this is a little ugly, but maybe we can fix this in a refactor at some point 😆